numpy/lib/tests/test_function_base.py

```diff
@@ -1407,6 +1447,52 @@ def test_size_zero_output(self):
         with assert_raises_regex(ValueError, 'new output dimensions'):
             f(x)
 
+    def test_positional_regression_9477(self):
+        # This supplies the first keyword argument as a positional,
+        # to ensure that they are still properly forwarded after the
+        # enhancement for #9477
+        f = vectorize((lambda x: x), ['float64'])
+        r = f([2])
+        assert_equal(r.dtype, np.dtype('float64'))
+
+    def test_subclass_regression_9477(self):
+        # Test that subclasses written prior to #9477 still work;
+        # these can be easily broken by a poor implementation of __new__.
+        arg_value = 'added arg!'
+        kw_value = 'added kw!'
+        call_value = 'added arg to call!'
+        class my_vectorize(vectorize):
+            # user can add args and keywords to __init__
+            def __init__(self, f, new_arg, new_kw=None, *args, **kwargs):
```

Comment:
Try again passing them as kwargs. You'll end up with a decorator when you expected an object